webpack/scenes/ModuleStreams/Details/Profiles/ProfileRpmsCellFormatter.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from 'patternfly-react';
+
+import './ProfileRpmsCellFormatter.scss';
+
+class ProfileRpmsCellFormatter extends Component {
+  constructor(props) {
+    super(props);
+    this.minAmount = 10;
```

Comment:
const in the class scope (before the constructor) makes more sense IMO

Reply:
I'll try, but looks like [this may not be possible](https://stackoverflow.com/questions/22528967/es6-class-variable-alternatives). The accepted answer suggests using the constructor. (javascript is fun)

Reply:
It's fine as is. My JS knowledge is lacking :)